### DISCUSSION TITLE
[Feature] 图标包

### DISCUSSION THREAD

**Discussion by abbeyarrasmithn** [gkd-kit/gkd#756] [Announcements]
### 新功能描述

作为一个强迫症，我强烈希望作者大大能够把应用图标中的那个篮球中的两条竖线换成与水平线垂直的两条标准竖线，或者提供一些新的图标包以供选择，谢谢。

**Comment by lisonge** [comment]
> 提供一些新的图标包

你可以自己制作手机主题

---

另外，那两根竖线不仅仅是🏀的线，还是坤坤的背带

![image](https://github.com/user-attachments/assets/dfbc65fa-c418-4f51-9eed-8e8eefcc21cc)
![image](https://github.com/user-attachments/assets/5cce8a75-8fea-4f0c-8a29-ac832a3eab4f)
